docs/5.x/pack.md

```diff
@@ -631,6 +632,15 @@ hooks:
 
   # restores application state from backup
   restore:
+
+  # install a custom CNI network plugin during cluster installation
```

Comment:
Would it make sense to put more detail about properly installing a CNI plugin or leave a reference to another source where it's described?

Reply:
I'm sort of under playing it in the docs, since we're opening up access, but it isn't a configuration we're willing to offer support on.